### PR TITLE
fix(delete): #MC-123 display booking deletion checkbox

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes
fix #83 :
- check booking deletion rights
- changes in popup appearance

## Checklist tests
for many cases (author, shared with RBS manage right, shared with no RBS right ...):
calendar view : click on created event with booking and delete it, the message with choice appears in the confirmation popup
list view:

try to delete one event without booking : behaviour is same as before
try to delete one event with booking : behaviour is same as calendar view + the event edition window opens
try to delete many events including one with booking: message states that the deletion will not impact [[event with booking's name]]'s booking.

## Checklist before requesting a review (magic string, indentation, comment/documentation...)
#MC-123 https://entsupport.gdapublic.fr/browse/MC-123

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -38,36 +38,36 @@ clean () {
 
 buildNode () {
   #jenkins
-  echo "[buildNode] Get branch name from jenkins env..."
-  BRANCH_NAME=`echo $GIT_BRANCH | sed -e "s|origin/||g"`
```
Comment (alicedraillard) [inline build.sh:42]: commits will be squashed before merging

File: src/main/resources/public/ts/directives/rbs-booking-delete-info/rbs-booking-delete-info.ts [hunk 2]
```diff
@@ -53,6 +53,18 @@ class Controller implements ng.IController, IViewModel {
         return this.display.list && this.hasListViewSelectedCalendarEventsWithBooking();
     }
 
+    /**
+     * Returns true if the user has the right to delete a booking
+     */
+    canDeleteBooking = () => {
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/rbs-booking-delete-info/rbs-booking-delete-info.ts:59]: typage

File: src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts [hunk 3]
```diff
@@ -101,5 +101,42 @@ export class RbsEmitter {
         return false;
     };
 
+    userIsResourceOwner = (): boolean => {
+        let rbsSnipletVm: any = this.getRbsSnipletController()['vm'];
+        if (this.scope.ENABLE_RBS && !!this.getRbsSnipletController() && !!rbsSnipletVm
+            && !!rbsSnipletVm.bookings && !!rbsSnipletVm.bookings.all) {
+            rbsSnipletVm.bookings.all.find(b => b.resource && (b.resource.owner == model.me.userId));
```
Comment (BastienBenard) [inline src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts:108]: typage de b

File: src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts [hunk 4]
```diff
@@ -101,5 +101,42 @@ export class RbsEmitter {
...
+        }
+        return false;
+    }
+
+    userIsAdmlForStructure = (): boolean => {
+        let rbsSnipletVm: any = this.getRbsSnipletController()['vm'];
+        if (this.scope.ENABLE_RBS && !!this.getRbsSnipletController() && !!rbsSnipletVm
+            && !!rbsSnipletVm.bookings && !!rbsSnipletVm.bookings.all && model.me.functions.ADMIN_LOCAL) {
+            let resourceTypeStructures: Array<String> = [];
+            rbsSnipletVm.bookings.all.forEach(b => {
```
Comment (BastienBenard) [inline src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts:120]: typage de b

File: src/main/resources/public/ts/directives/rbs-booking-delete-info/rbs-booking-delete-info.ts [hunk 5]
```diff
@@ -49,10 +49,22 @@ class Controller implements ng.IController, IViewModel {
...
      */
-    hasBookingsToDeleteListView = () => {
+    hasBookingsToDeleteListView = (): boolean => {
         return this.display.list && this.hasListViewSelectedCalendarEventsWithBooking();
     }
 
+    /**
+     * Returns true if the user has the right to delete a booking
+     */
+    canDeleteBooking = ():boolean => {
```
Comment (JinDouang) [inline src/main/resources/public/ts/directives/rbs-booking-delete-info/rbs-booking-delete-info.ts:59]: indentation

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts [hunk 6]
```diff
@@ -101,5 +101,37 @@ export class RbsEmitter {
...
+            return rbsSnipletVm.bookings.all.find((b: Booking) => b.resource && (b.resource.owner == model.me.userId));
+        }
+        return false;
+    }
+
+    userIsAdmlForStructure = (): boolean => {
+        let rbsSnipletVm: any = this.getRbsSnipletController()['vm'];
+        if (this.scope.ENABLE_RBS && !!this.getRbsSnipletController() && !!rbsSnipletVm
+            && !!rbsSnipletVm.bookings && !!rbsSnipletVm.bookings.all && model.me.functions.ADMIN_LOCAL) {
+            let resourceTypeStructures: Array<String> = [];
```
Comment (JinDouang) [inline src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts:117]: String => string

**Review by JinDouang — CHANGES_REQUESTED** [review]

**Pull request merged**